src/users/models.ts

```diff
@@ -0,0 +1,41 @@
+import { db } from '../config'
+
+
+const TABLE_NAME = 'users'
+
+const USER_FIELDS = [
+  'id', 'fullName', 'email', 'phoneNumber', 'canVote', 'profileUrl', 'studentId'
```

Comment:
Creo que aquí debería estar el Telegram Handle también

```suggestion
  'id', 'fullName', 'email', 'phoneNumber', 'telegramHandle', 'canVote', 'profileUrl', 'studentId'
```